### ISSUE TITLE
sadly not working

### ISSUE THREAD

**Issue by ClaraCrazy** [ProgrammerIn-wonderland/hmfullpy#1]
Guessing pypi is just being a pain in the ass but..
![image](https://cloud.cynthialabs.net/s/peiMqH7aLxfn5aA/preview)

clearly not installing properly. shame

**Comment by ProgrammerIn-wonderland** [comment]
This is actually 100% my fault because I couldn't figure out how to set it up with pypi, some directions from anyone would be greatly appreciated lol. The online guide I followed was terrible

**Comment by ClaraCrazy** [comment]
maybe that helps?
https://medium.com/@joel.barmettler/how-to-upload-your-python-package-to-pypi-65edc5fe9c56

**Comment by VyZi** [comment]
It works perfectly

As it isn't on PyPi yet just run your script in the src folder and it should work

About getting the thing to PyPi I'm not sure how to do that as I've never done it before

**Comment by ProgrammerIn-wonderland** [comment]
Yeah for now the way to run it is to just put it in your src folder unfortunately, once I get school over with I'll see how to use pypi

**Comment by ProgrammerIn-wonderland** [comment]
I'm so sorry, school has been taking up alot of time, I'll try and get it on next weekend-ish

**Comment by BlueBerrySans365** [comment]
Do you need help?
